packages/filesystem/src/electron-browser/file-dialog/electron-file-dialog-service.ts

```diff
@@ -24,6 +24,11 @@ import { FileStat } from '../../common';
 import { FileAccess } from '../../common/filesystem';
 import { DefaultFileDialogService, OpenFileDialogProps, SaveFileDialogProps } from '../../browser/file-dialog';
 
+// Copied from https://github.com/electron/electron/blob/v4.2.12/docs/api/dialog.md
```

Comment:
@thegecko we will require a [CQ](https://github.com/eclipse-theia/theia/wiki/Registering-CQs) for the copied code, unless it's been previously covered somehow.